config.js

```diff
@@ -1,15 +1,10 @@
 const config = {
-	redisUrl: process.env.STETHOSCOPE_REDIS_URL,
-	redisReconnectionDelay: 0,
-	keyTtl: 86400,
-
-	// event keys
-	eventKeyDelimiter: ':',
-	ageEventKeyPrefix: 'AGE',
+	dbURI: process.env.STETHOSCOPE_MONGO_URI,
+	eventTTL: process.env.STETHOSCOPE_EVENT_TTL || 86400
 };
 
 if (process.env.NODE_ENV !== 'production') {
-	config.redisUrl = 'redis://127.0.0.1:6379/0';
+	config.dbURI = 'mongodb://localhost:27017/ft-email-stethoscope-test';
```

Comment:
Could we remove this `if`-statement and do like this instead:
```suggestion
	config.dbURI = process.env.STETHOSCOPE_MONGO_URI || 'mongodb://localhost:27017/ft-email-stethoscope-test';
```

Reply:
Ah, good idea! I applied your suggested patch 👍

Reply:
Ah, nifty! But could we move the `|| 'mongodb[…]'` default value to the config-block above? Sorry to be a bother!

Reply:
Ah, my bad, it should be fixed now 🙇